backend/migrations/dataall_migrations/migrationmanager.py

```diff
@@ -0,0 +1,65 @@
+import os
+from collections import deque
+from typing import Deque
+from migrations.dataall_migrations.versions.initial import InitMigration
+from migrations.dataall_migrations.base_migration import MigrationBase
+
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
+
+
+class MigrationManager:
+    def __init__(self, key='0', initial_migration=InitMigration):
+        self.current_migration = initial_migration
+        self.previous_migrations: Deque[MigrationBase] = deque()
+        while self.current_migration.revision_id() != key:
```

Comment:
if `key` doesn't exist this will blow up ungracefully I think `AttributeError: 'NoneType' object has no attribute 'revision_id'`